Tests/test_models_accuracy.py

```diff
@@ -1,9 +1,13 @@
 import numpy as np
 import pandas as pd
 import pytest
-from pyprocessmacro import Process
+import sys
```

Comment:
@BillmanH is this necessary?

Reply:
Well from the file in it's location I wasn't able to import from another folder with no upstream module. This way it ran without having to change the entire structure of the application, which I was hesitant to do. LMK if you think it should be done in a different way.